util/telemetry/src/lib.rs

```diff
@@ -2,19 +2,34 @@
 
 //! OpenTelemetry wrappers and helper utilities.
 
+use opentelemetry::{
+    global::BoxedTracer,
+    sdk::{trace::Config, Resource},
+};
```

Comment:
This should be below the `pub use` statement, and separated by newline, otherwise CLion's "add import" auto-fixer isn't particularly great at figuring out where to put this stuff.